### ISSUE TITLE
Dropdowns not readable in dark mode

### ISSUE THREAD

**Issue by nbarry10** [unl/wdntemplates#1457]
![drop-down-dark-mode](https://user-images.githubusercontent.com/5738813/91899345-a22c8c00-ec62-11ea-90da-996626367d90.png)

**Comment by skoolbus39** [comment]
`<select>` dropdowns should be readable now:

<img width="495" alt="Screen Shot 2021-09-14 at 4 07 45 PM" src="https://user-images.githubusercontent.com/5385125/133334097-e4e9c552-04eb-4e18-81a7-4da780477d26.png">
